docker/container.go

```diff
@@ -62,23 +62,23 @@ func (c *ContainerService) ContainerCreate(ctx context.Context, config *containe
 	if strings.Contains(ctn, "notfound") &&
 		!strings.Contains(ctn, "ignorenotfound") {
 		return container.ContainerCreateCreatedBody{},
-			errdefs.NotFound(fmt.Errorf("Error: No such container: %s", ctn))
+			errdefs.NotFound(fmt.Errorf("error: No such container: %s", ctn))
```

Comment:
Do we want to do this?

I understand what the linter service is complaining about with capitalized error messages.

And I actually agree with the linter that error messages shouldn't be capitalized like this 👍 

However, the reason these error messages are capitalized is due to Docker returning the messages like this 😢 

When I submitted this code originally, I briefly touched on this:

https://github.com/go-vela/mock/pull/40

Reply:
Perfect, didn't know that. This is where `nolint` with comment/explanation comes in handy.

Reply:
Fixed. Thanks.